pricepoint_intel/geospatial/risk_framework.py

```
@@ -0,0 +1,1076 @@
+        """Calculate price volatility for a SKU over time.
+
+        Args:
+            sku_id: SKU identifier
+            vendor_id: Optional vendor filter
+            days: Number of days to analyze
+
+        Returns:
+            Dictionary with volatility metrics
+        """
+        with session_scope(self.db_config) as session:
+            cutoff = datetime.utcnow()
```

Ruff (0.14.6)

Local variable `cutoff` is assigned to but never used

Remove assignment to unused variable `cutoff`

(F841)